docs/user/resources/06-20-docker-registry-cr.md

```diff
@@ -52,18 +58,21 @@ For details, see the [Docker Registry specification file](https://github.com/kym
 
 **Spec:**
 
-| Parameter                               | Type   | Description                                                                             |
-|-----------------------------------------|--------|-----------------------------------------------------------------------------------------|
-| **storage**                             | object | Contains configuration of the registry images storage.                                  |
-| **storage.azure**                       | object | Contains configuration of the Azure storage.                                            |
-| **storage.azure.secretName** (required) | string | Specifies the name of the Secret that contains data needed to connect to the Azure storage. |
-| **storage.s3**                          | object | Contains configuration of the s3 storage.                                               |
-| **storage.s3.bucket** (required)        | string | Specifies the name of the s3 bucket.                                                    |
-| **storage.s3.region** (required)        | string | Specifies the region of the s3 bucket.                                                  |
-| **storage.s3.regionEndpoint**           | string | Specifies the endpoint of the s3 region.                                                |
-| **storage.s3.encrypt**                  | string | Specifies if data in the bucket is encrypted.                                      |
-| **storage.s3.secure**                   | string | Specifies if registry uses the TLS communication with the s3.                           |
-| **storage.s3.secretName**               | string | Specifies the name of the Secret that contains data needed to connect to the s3 storage.    |
+| Parameter                               | Type   | Description                                                                                                   |
+|-----------------------------------------|--------|---------------------------------------------------------------------------------------------------------------|
+| **externalAccess**                      | object | Contains configuration of the registry external access throw the kyma-gateway Istio Gateway with tls enabled. |
```

Comment:
I have trouble understanding the description for **externalAccess**. Can you elaborate?